src/iframe/src/Iframe.js

```diff
@@ -930,7 +954,11 @@ class Iframe extends Component {
             width={CANVAS_SIZE}
             height={CANVAS_SIZE}
             ref={_canvas => {
-              this._canvas = _canvas
+              if (_canvas) {
```

Comment:
I'm not sure we should have anything in here besides setting the react ref. I get a feeling it's an antipattern. But not sure.

Reply:
Fixed.